### DISCUSSION TITLE
For PHP-server???

### DISCUSSION THREAD

**Discussion by ivose** [eclipse-theia/theia#8584] [Ideas] [answered]
Is it possible to add theia into a PHP-pased webpage? Coding in a localhost's xampp-server and adding to php (cpanel-based) server throw ftp-client and testing there is uncomphortable.

**Comment by paul-marechal** [comment, marked as answer]
Not sure about what you have in mind, but a Theia app is composed of a Node.js process (backend) that serves a whole web app (frontend). I don't know if it would be to run the whole frontend inside some iframe on the page served by PHP, but you'd still need to run the backend process somehow.
